src/js/messaging/reducers/folders.js

```diff
@@ -21,7 +22,8 @@ const initialState = {
   },
   ui: {
     nav: {
-      expanded: false
+      expanded: false,
```

Comment:
Maybe rename this to `foldersExpanded` or something similar. At a glance, it isn't clear what the differences between `expanded` and `visible` are.